Controller/Module/ActivityModuleController.php

```diff
@@ -300,12 +300,6 @@ public function editAction(Request $request, $id)
 
             $this->handler->postPersistEditEvent($this->operations[0], $form, $content);
 
-            if ($form->get('campaignchain_hook_campaignchain_due')->has('execution_choice') && $form->get('campaignchain_hook_campaignchain_due')->get('execution_choice')->getData() == 'now') {
```

Comment:
Yes. That's a bug I found while working on this. Guess, I should have created a separate issue to avoid confusion.